test/fail_compilation/fob1.d

```diff
@@ -17,6 +17,7 @@ fail_compilation/fob1.d(104): Error: variable `fob1.foo1.p` is returned but is U
 
 /* TEST_OUTPUT:
 ---
+fail_compilation/fob1.d(204): Error: variable `fob1.foo2.p` assigning to Owner without disposing of owned value
```

Comment:
It's weird that a null pointer has to be disposed. Is this really what we want?

Reply:
It's also weird that a null pointer can be left dangling, but currently `null` is seen equally as any other pointer.